pkg/registry/common/localbypass/server.go

```diff
@@ -21,42 +21,56 @@ package localbypass
 import (
 	"context"
 	"net/url"
+	"sync"
 
-	"github.com/networkservicemesh/sdk/pkg/tools/localbypass"
+	"github.com/networkservicemesh/sdk/pkg/registry/common/custom"
 
-	"github.com/golang/protobuf/ptypes/empty"
 	"github.com/networkservicemesh/api/pkg/api/registry"
-
-	"github.com/networkservicemesh/sdk/pkg/registry/core/next"
 )
 
-type localBypassRegistry struct {
-	sockets localbypass.SocketMap
+// LocalBypass - return interafce to return localbypass endpoints URIs
+type LocalBypass interface {
+	LoadEndpoint(endpointName string) (*url.URL, bool)
```

Comment:
[minor] Should we rename it to `LoadPass` ?

Reply:
Thanks.